requirements/fabric/test.txt

```diff
@@ -7,4 +7,3 @@ pytest-rerunfailures ==12.0
 pytest-random-order ==1.1.0
 click ==8.1.7
 tensorboardX >=2.2, <2.7.0  # min version is set by torch.onnx missing attribute
-torchmetrics >=0.7.0, <1.5.0 # needed for using fixed compare_version
```

Comment:
Seems like this only removes it from the test requirements?

Reply:
yes, I was thinking about removing it also from PT but that would be a breaking change so we would need to do it later...
@lantiga or is fine to drop it also from PL where we had it for a long time as a compatibility patch since we cut out Tm to the separate package?